web_src/less/_base.less

```diff
@@ -774,6 +774,7 @@ footer {
     .container {
         width: 100vw !important;
         padding: 0 0.5rem;
+        max-width: calc(100vw - 1rem) !important;
```

Comment:
Back to `!important` are we, isn't it? 🤣

Reply:
Yeah, it's needed because it is overriding a style from fomantic:

````
.ui.container {
    max-width: 100%;
}
````